plugins/modules/ndo_dhcp_option_policy.py

```diff
@@ -98,6 +98,16 @@
     state: query
   register: query_one
 
+- name: Query a dhcp option policy with UUID
+  cisco.mso.ndo_dhcp_option_policy:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_tenant_template
+    option_policy_uuid: '{{ query_one.current.uuid }}'
```

Comment:
Can you please change '{{ query_one.current.uuid }}' to '{{ create.current.uuid }}'  only in the examples section of all the modules? I think it's better to reference the uuid with creation rather than query.